src/applications/letters/containers/LetterList.jsx

```diff
@@ -136,6 +136,11 @@ export class LetterList extends React.Component {
           through these links:
         </p>
         <ul>
+          <li>
+            <a href="/education/download-letters/" target="_blank">
+              <strong>Download your VA education letters.</strong>
+            </a>
+          </li>
```

Comment:
Just curious where this came from? I don't see it mentioned in the linked issue, was this from a side conversation?

Reply:
Awesome, thank you!